### ISSUE TITLE
Vento template output not updated when using Eleventy dev server

### ISSUE THREAD

**Issue by carpelli** [noelforte/eleventy-plugin-vento#34]
#### Steps to reproduce
1. Run `npx @11ty/eleventy --serve`
2. Modify a watched Vento template file

The console output will be something like:
```
[11ty] File changed: index.vto
[11ty] Writing ./_site/index.html from ./index.vto
[11ty] Wrote 1 file in 0.01 seconds (v3.0.0)
[11ty] Watching…
```
####  Expected behavior
The template is rerendered and written to the output directory

#### Actual behavior
The output file is modified but not updated to reflect the change in the template

Any errors in the template aren't reported either, so I suppose it the updated template is never rendered

**Comment by noelforte** [comment]
Shipped in 3.0.2, thanks for catching this one!

**Comment by carpelli** [comment]
Thanks for the fix and the plugin itself!
